app/_hub/kong-inc/key-auth/overview/_index.md

```diff
@@ -287,19 +295,55 @@ Response:
 }
 ```
 
+{% if_version gte: 3.10.x %}
+
+## Configure realms for centralized Consumers in {{site.konnect_short_name}}
```

Comment:
Should this be an [example](https://deploy-preview-7657--kongdocs.netlify.app/hub/kong-inc/key-auth/unreleased/how-to/basic-example/) rather than on the overview?

Reply:
Good point, I'll convert it to an example.